### DISCUSSION TITLE
NeMo public roadmap

### DISCUSSION THREAD

**Discussion by adiIspas** [NVIDIA-NeMo/NeMo#1682] [General]
Hello, awesome people!

First of all, thank you very much for the job that you do with this toolkit.

Secondly, do you have a public roadmap for NeMo toolkit? I think will be useful for everyone that uses NeMo to know when new version/features will come.

In my case, I'm interested to know when will be available support for words timestamps and confidence with language model.

Regards,
Adrian

**Comment by okuchaiev** [comment]
Hi @adiIspas,

This is a good question. First, let me mention that:
* NeMo is being developed in the open. E.g. what you see in the "main" branch on GH is what our team members use in their daily work. In this sense, things which are now WIP in the main branch will be appearing as features in our next releases.
* We accept external contributions, so if you'd like to send a PR to us, please have a look here: https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md 

Here is what we can share right now:

* we plan to release "1.0.0rc" version soon (this winter) which will have some configuration improvements, ASR models in few more languages (using Mozilla's Common Voice), machine translation models and other new features and bugfixes.
* in about a month after that we'll release "1.0.0" which shouldn't differ much from "1.0.0rc" in terms of features and usage but should provide more stability and overall performance.

We'll think about sharing a full roadmap (or a subset of it) publicly.

**Reply by okuchaiev** [reply]
Quick update: The "1.0.0rc1" was released yesterday. Please check release notes for what is new (A LOT).
This is a "release candidate" update and we are hoping to release 1.0.0 towards the end of this month (April)
